AlphaWallet/Tokens/ViewModels/TokensViewModel.swift

```diff
@@ -17,7 +17,7 @@ class TokensViewModel {
             totalAmount += amount(for: token)
         }
         guard totalAmount != 0 else { return "--" }
-        return CurrencyFormatter.formatter.string(from: NSNumber(value: totalAmount))
+        return NumberFormatter.currency.string(from: totalAmount)
```

Comment:
It looks like the `amount` property is not used anymore. Shall we remove it?

Reply:
seems like we can to remove it.